Date Time editor modification

serjiokov opened the pull request:
Was added new widget editor for date, time, timestamp data types.

AndrewKhitrin committed:
DebugManager connection

Commit:
#2556 extract breakpoints to the separate package

Commit:
#2556 extract DBGController to the separate package

Commit:
Move database debug controller

Commit:
#2556 split i18n resources

Commit:
#2556 replace IProgressMonitor with DBRProgressMonitor

AndrewKhitrin committed:
Before migration

AndrewKhitrin committed:
Merge branch 'devel' into 2556-postgres-debug

AndrewKhitrin committed:
2556-postgres-debug after merge

Commit:
#2556 remove eclipse types from DBGController

Commit:
#2556 move procedure debug model id to debug.core

Commit:
#2554 move procedure debug model presentation to debug.ui

Commit:
#2556 OK, capitalize the "UI"

Commit:
#2556 move ATTR_OID to debug.core

Commit:
#2556 move error reporting utils to debug.core

Commit:
#2556 add ProcedureDebugController to abstract from PG SQL

Commit:
#2556 abstract ProcedureThread from PG SQL

Commit:
#2556 rename thread label to DB-neutral

AndrewKhitrin committed:
Attach worked !

liuyuanyuan committed:
Modify l10n for copy special。

Commit:
#2556 generalize PgSqlDebugTarget to ProcedurDebugTarget

serjiokov committed:
Date Time editor modification

Was added new widget editor for date, time, timestamp data types.

AndrewKhitrin committed:
Simple case worked

serge-rider committed:
Navigator utils additions. Object browser fix

serge-rider committed:
#1712 PG permissions management UI

serge-rider committed:
#1712 PG permissions management UI

serge-rider committed:
#1712 PG role manager. i18n fix.

serge-rider committed:
New object create - error handling

AndrewKhitrin committed:
All steps worked

serge-rider committed:
#1712 PG role manager + editable role

AndrewKhitrin committed:
Merge branch 'devel' into 2556-postgres-debug

serjiokov committed:
change accesses modificator.

serge-rider commented:
Visible issues:
- On Windows initial timestamp controls layout is still broken (day of month is not visible)
- Edit control is too wide (gets over the screen edge if timestamp column is the last one). Perhaps time editor should be under date editor
- LONG date format isn't convenient (it is too different from value in grid). MEDIUM is more suitable.
- Panel editor shouldn't fill vertically (it looks ugly in value view panel).
- We still need to support plain-text editor (as it is done now). Many people prefer this editor over fancy UI.

serge-rider commented:
Also: no need to declare formatter constants in CustomTimeEditor. They are already declared in DBDDataFormatter.

serge-rider committed:
#2725 Grid cell link rendering fix

serge-rider committed:
#1712 PG role create/drop

efradkin committed:
#2725 Foreign key link icon render fix

beonit committed:
"Export resultset as Json" use column label instead of column name.

Fixed column label specified as "AS" to be output to json export.
Json export use column label instead of column name.

Query : SELECT CTGRY_ID AS categoryId FROM TSPC_CTGRY LIMIT 2;
Old result : [{"CTGRY_ID" : 1001}, {"CTGRY_ID" : 1002}]
Change result : [{"categoryId" : 1001}, {"categoryId" : 1002}]
fixed : [{"categoryId" : 1001}, {"categoryId" : 1002}]

Commit:
#2335 constructor and method declaration brace to end_of_line

Commit:
Merge pull request #2729 from liuyuanyuan/devel

Modify l10n for copy advance

serge-rider committed:
Merge pull request #2731 from beonit/patch-1

"Export resultset as Json" use column label instead of column name.

serge-rider committed:
#2731 Check for empty label

serge-rider committed:
#2712 SQL Server/Sybase schema filter fix

serge-rider committed:
Objects browser dialog UI fix (add filter)

serge-rider committed:
Objects browser dialog UI fix (add filter)

serjiokov committed:
Changed String constants by DBDDataFormatter type.

serge-rider committed:
#2639 Open multiple selected objects on Enter

AndrewKhitrin committed:
Variables, Code format, Session termination, Some details

serge-rider committed:
#1712 PG privilege editor redesign (use nav tree)

AndrewKhitrin committed:
Broken Deps

serge-rider committed:
Merge pull request #2734 from dbeaver/2556-postgres-debug

2556 postgres debug

Commit:
Merge pull request #2735 from dbeaver/2556-postgres-to-procedure

2556 postgres to procedure

serge-rider committed:
Merge pull request #2727 from dbeaver/#2725

#2725 Foreign key link icon render fix

serge-rider committed:
#1712 PG privilege editor redesign (use nav tree)

serge-rider committed:
#1712 PG privilege editor redesign (use nav tree)

serge-rider committed:
Debugger model refactoring

serge-rider committed:
Debugger model refactoring

serge-rider committed:
#1712 PG permissions: grant/revoke command

serge-rider committed:
#1712 PG grant/revoke command

serge-rider committed:
Entity editor save fix (empty command chain)

serge-rider committed:
#2585 PG table column comments in DDL (for existing columns)

serge-rider committed:
#1712 PG grant/revoke command, permission management fixes

serge-rider committed:
Navigator tree default selection fix

serge-rider committed:
#1712 Batch objects processing support

serge-rider committed:
#1712 Permission table enablement. Batch objects processing support

serge-rider committed:
#1712 Permission panel refresh fix

serge-rider committed:
Navigator tree loading visualizer fix (root node isn't DBNRoot)

serge-rider committed:
Navigator tree filtering enhancement (users & roles)

Commit:
#2556 reuse DBRResult for PostgreDebugSession

Commit:
#2556 add extension point to register procedure controllers

Commit:
#2556 restore shortcut contribution

Commit:
#2556 move ProcedureLaunchDelegate to dbeaver.debug.core

serge-rider committed:
Pull ds provider ID in driver interface

serge-rider committed:
Pull ds provider ID in driver interface

Commit:
#2556 move PostgreProcedureController to platform-less part

Commit:
Merge pull request #2743 from dbeaver/2556-postgre-integrate-console

2556 postgre integrate console

serge-rider committed:
Schema gen test fix

serge-rider committed:
#2656 Statement execute fix - limit apply check (workaround for SQL Server)

serge-rider committed:
Merge remote-tracking branch 'origin/devel' into devel

serge-rider committed:
#2739 DB2 connection page fix

serge-rider committed:
Connection pages fix (settings load/save)

serjiokov committed:
Reworked to medium style.
Since now the Time editor widget under the Date editor.

serjiokov committed:
Formated code

Commit:
#2556 add IStatus handling to Log

Commit:
#2556 remove DBRResult from postgresql.debug

Commit:
#2556 remove DBRResult

Commit:
#2556 remove DBRResult

Commit:
#2556 PostgreDebugSessionWorker.sql should be final

Commit:
Merge pull request #2745 from dbeaver/2556-postgre-steps

2556 postgre steps

Commit:
#2556 create DBGSessionManager

serge-rider committed:
4.3.2 config

serge-rider committed:
String content implementation

Commit:
#2556 bind session create/close

AndrewKhitrin committed:
Variables set

AndrewKhitrin committed:
Merge branch '2556-postgre-steps' of https://github.com/dbeaver/dbeaver into 2556-postgre-steps

serge-rider committed:
Support string content in content editor

serge-rider committed:
#2584 Open string in a separate editor instead of dialog (optional)

serge-rider committed:
Merge pull request #2746 from dbeaver/2556-postgre-steps

2556 postgre steps

serge-rider committed:
PG procedure OID property

serge-rider committed:
Debug API refactoring

serge-rider committed:
Ref table context menu shortcut fix

serge-rider committed:
PG role editor fix

AndrewKhitrin committed:
Debugger attach to procedure

AndrewKhitrin committed:
Merge branch 'devel' of https://github.com/dbeaver/dbeaver into devel

Commit:
#2556 comment out the call

Commit:
#2556 DBGObject to "Object getID()"

Commit:
#2556 DBGSessionInfo to "Object getID()"

Commit:
#2556 replace id parameters with java.lang.Object

Commit:
#2556 remove SESSION_INFO extends DBGSessionInfo from template

Commit:
#2556 add toMap() for DBG beans

Commit:
#2556 remove <DEBUG_OBJECT extends DBGObject> from DBGSession

Commit:
#2556 move ResourceSet to try-with-resources

Commit:
#2556 replace extension point with adapter

Commit:
#2556 rename to ToggleProcedureBreakpointTarget

serge-rider committed:
Russian localization

serge-rider committed:
Icons fix

serge-rider committed:
Merge remote-tracking branch 'origin/devel' into devel

serge-rider committed:
Disable debugger plugins

AndrewKhitrin committed:
Typical scenario for PostreDebugSession

Commit:
#2556 avoid using internals

Commit:
#2556 extract contribution constants

Commit:
#2556 override context menu id for editor and ruler

Commit:
#2556 add "Toggle Breakpoint" to the ruler menu

Commit:
#2556 merge PgSqlTab into DatabaseTab

serge-rider committed:
Merge pull request #2747 from dbeaver/2556-postgres-generics

2556 postgres generics

serge-rider committed:
Code reformat

serge-rider committed:
#2749 SQL Server/Sybase conn page: port is optional

serge-rider committed:
#2750 SQL Server: schema list read fix

serjiokov committed:
Merge branch '2254_date_time_editor' into devel

serjiokov committed:
Added new preferences section for support DateTime Editor and String
Editor for Date, Time, Timestamp types.

Added support in editor part.
